relay-event-schema/src/protocol/span.rs

```diff
@@ -931,4 +931,38 @@ mod tests {
         assert_eq!(data.get_value("code\\.namespace"), Some(Val::String("ns")));
         assert_eq!(data.get_value("unknown"), None);
     }
+
+    #[test]
+    fn test_span_data_empty_well_known_field() {
+        let span = r#"{
+            "data": {
+                "lcp.url": ""
+            }
+        }"#;
+        let span: Annotated<Span> = Annotated::from_json(span).unwrap();
+        assert_eq!(span.to_json().unwrap(), r#"{"data":{"lcp.url":""}}"#);
+    }
+
+    #[test]
+    fn test_span_data_empty_custom_field() {
+        let span = r#"{
+            "data": {
+                "custom_field_empty": ""
+            }
+        }"#;
+        let span: Annotated<Span> = Annotated::from_json(span).unwrap();
+        assert_eq!(
+            span.to_json().unwrap(),
+            r#"{"data":{"custom_field_empty":""}}"#
+        );
+    }
+
+    #[test]
+    fn test_span_data_completely_empty() {
+        let span = r#"{
+            "data": {}
+        }"#;
+        let span: Annotated<Span> = Annotated::from_json(span).unwrap();
+        assert_eq!(span.to_json().unwrap(), r#"{"data":{}}"#);
+    }
```

Comment:
I verified that this test has the same result on `master`. `skip_serialization` applies to child elements, not the element itself.